src/module/settings.js

```diff
@@ -73,6 +73,15 @@ export function registerSystemSettings() {
     default: false,
```

Comment:
Suggesting `requiresReload = true` so the combat-tracker is refreshed and the lock icon appear/disappear properly.
```suggestion
    default: false,
    requiresReload: true,
```